ibis/backends/flink/tests/test_ddl.py

```diff
@@ -117,6 +121,145 @@ def test_create_table(con, awards_players_schema, temp_table, csv_source_configs
     assert temp_table not in con.list_tables()
 
 
+def test_recreate_table_from_schema(
+    con, awards_players_schema, temp_table, csv_source_configs
+):
+    # create table once
+    new_table = con.create_table(
+        temp_table,
+        schema=awards_players_schema,
+        tbl_properties=csv_source_configs("awards_players"),
+    )
+    assert temp_table in con.list_tables()
+    assert new_table.schema() == awards_players_schema
+
+    # create the same table a second time should fail
+    with pytest.raises(
+        Py4JJavaError,
+        match="org.apache.flink.table.catalog.exceptions.TableAlreadyExistException",
+    ):
+        new_table = con.create_table(
+            temp_table,
+            schema=awards_players_schema,
+            tbl_properties=csv_source_configs("awards_players"),
+            overwrite=False,
+        )
+
+
+def test_force_recreate_table_from_schema(
+    con, awards_players_schema, temp_table, csv_source_configs
+):
+    # create table once
+    new_table = con.create_table(
+        temp_table,
+        schema=awards_players_schema,
+        tbl_properties=csv_source_configs("awards_players"),
+    )
+    assert temp_table in con.list_tables()
+    assert new_table.schema() == awards_players_schema
+
+    # force creating the same twice a second time
+    new_table = con.create_table(
+        temp_table,
+        schema=awards_players_schema,
+        tbl_properties=csv_source_configs("awards_players"),
+        overwrite=True,
+    )
+    assert temp_table in con.list_tables()
+    assert new_table.schema() == awards_players_schema
+
+
+@pytest.mark.parametrize(
+    "employee_df",
+    [
+        pd.DataFrame(
+            [("fred flintstone", "award", 2002, "lg_id", "tie", "this is a note")]
+        )
+    ],
+)
+@pytest.mark.parametrize(
+    "schema_props", [(None, None), (_awards_players_schema, "awards_players")]
+)
+def test_recreate_in_mem_table(
+    con, employee_df, schema_props, temp_table, csv_source_configs
+):
+    # create table once
+    schema = schema_props[0]
+    if schema_props[1] is not None:
+        tbl_properties = csv_source_configs(schema_props[1])
+    else:
+        tbl_properties = None
```

Comment:
```suggestion
@pytest.mark.parametrize(
    "schema, table_name", [(None, None), (_awards_players_schema, "awards_players")]
)
def test_recreate_in_mem_table(
    con, employee_df, schema, table_name, temp_table, csv_source_configs
):
    # create table once
    if table_name is not None:
        tbl_properties = csv_source_configs(table_name)
    else:
        tbl_properties = None
```

only a suggestion -- I always have trouble remembering what the various elements of a container are supposed to represent without names